problem2/problem2.rb

```diff
@@ -0,0 +1,23 @@
+class Problem2
+	attr_accessor :result,:array
+
+	def  initialize(*args)
+		@array = args;
+	end
+
+	def Largest2Numbers
```

Comment:
How about returning the result via the method? storying it in a instance variable makes you implementation more complicated, state mutation is usually not a good idea.

Your function should try to be [pure](https://en.wikipedia.org/wiki/Pure_function)